client/src/index.js

```diff
@@ -1,11 +1,19 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
-import App from './App';
+// import App from './App';
+
+import Sidebar from './components/Sidebar';
+
+const TodaySchedule = () => <h3>Today Schedule</h3>;
+const Patients = () => <h3>Patients</h3>;
+const Calender = () => <h3>Calender</h3>;
+
+const options = [<TodaySchedule />, <Patients />, <Calender />];
 
 ReactDOM.render(
   <React.StrictMode>
-    <App />
+    <Sidebar contentComponents={options} />
```

Comment:
You should require this container at the `App.js` when we are at the `/dashboard` route

Reply:
Done